docs/config.toml

```diff
@@ -1,4 +1,4 @@
-baseURL = "https://edge.docs.radapp.io/"
+baseURL = "https://docs.radapp.io/"
```

Comment:
The edge configuration changes in `config.toml` and `docs/layouts/partials/hooks/body-end.html` should be excluded. Think we should exclude them from the upmerge workflow ?